apps/server/src/entities/learnroom/course.entity.ts

```diff
@@ -62,7 +62,7 @@ export class Course extends BaseEntityWithTimestamps {
 		Object.assign(this, {});
 	}
 
-	getStudentsNumber(): number {
+	getNumberOfStudents(): number {
 		// TODO remove "|| []" when we can rely on db schema integrity
 		return (this.studentIds || []).length;
```

Comment:
can this.studentIds || [] be defined in a separate getter and the attribute changed to be private? otherwise this would be repeated again and again... in the getter the value would no more be undefined but []. 
why not using collections here?

Reply:
collection (ManyToMany) should work fine. can we reference to the `User` entity here?

Reply:
i would go for the user entity, i don't like id's here at all.

Reply:
We have no working user entity. If you add this, we can use it.

Reply:
Hm, you should add it here just with the required props. Using Id's is not a solution to go with.

Reply:
exactly. we can develop the entities over time starting with the props we need right now.